.github/workflows/check-clippy-all-features.yml

```diff
@@ -0,0 +1,58 @@
+name: Check Clippy all-features
+
+on:
+  pull_request:
+
+jobs:
+  get-head-commit-msg:
```

Comment:
I forgot to mention this when reviewing the other PR. It's not nice that this pops up as its own job in the Actions view. Can we merge it with the actual step below, and bail out early if the condition is not met? The name of the job would remain the "main" one.

Reply:
Condition will be mentioned in each step yet yes this looks better